lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -6,7 +6,7 @@ class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmC
     expose :resize_confirm, :override_return => nil
     expose :resize_revert,  :override_return => nil
 
-    expose :validate_resize
+    expose :supports_resize?
```

Comment:
@durandom So should I keep the expose validate_resize statement as it is and change the validate_resize method body to internally use supports_resize?